pydoctor/model.py

```diff
@@ -353,18 +352,16 @@ def documentation_location(self) -> DocLocation:
         else:
             return DocLocation.OWN_PAGE
 
-    def setup(self):
+    def setup(self) -> None:
         super().setup()
-        self.all = None
+        self.all: Optional[Collection[str]] = None
```

Comment:
I think that it would help to add as a documentation comment all that you know/think so far that `all` should do.

I see that it is used in astbuilder and it looks like it contains the members define in the `__all__`` member of a module.

----------

Also, by reading the code it was not clear why we need a Package and a Module inside model.py

It looks like Package is some sort of "wrapper" for the __init__ module.

Reply:
I agree about more documentation, but that's a problem we'll have to solve bit by bit, since there is quite an amount of technical debt. That said, now is a better time than average to pay off this particular piece of debt, so I'll add a docstring.